frontend/app/modules/components/issue-input-directive.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use-strict';
+  angular
+    .module('issue-input')
+    .directive('issueInput', Directive);
+    
+    function Directive(){
+      function Controller() {
+        var vm = this;
+      }
+
+      return {
+        restrict: 'E',
+        templateUrl: 'issue-input-directive.html',
+        controller: [Controller],
+        controllerAs: 'vm',
+        bindToController: true,
+        scope: {
+          label: '=',
```

Comment:
Since label is more than often going to be a text string, you can set the scope setting to be `@` instead of `=`. `@` will evaluate the expression, so instead of putting `<issue-input-directive label="'fieldName'">`, you can put `<issue-input-directive label="fieldName">`. The difference is the lack of quotes around the second one - it'll automatically be considered a string, vs. the first one which would be considered a variable without the string quote.